studio/app/optinist/wrappers/caiman/cnmf.py

```diff
@@ -202,21 +215,45 @@ def caiman_cnmf(
             "table_name": "Background",
             "region": list(range(n_rois, n_rois + n_bg)),
             "name": "Background_Fluorescence_Response",
-            "data": cnm.estimates.f.T,
+            "data": cnm.estimates.f.T if cnm.estimates.f is not None else np.array([]),
+            "unit": "lumens",
+        },
+        "estimates.W": {
```

Comment:
@quanpython 
The following is a question, the size of the NWB output files for cnmf and cnmfe are very different, is this likely due to the data in this part of the process?

- Example
  - whole_caiman_cnmf.nwb ... 0.65 MB
  - whole_caiman_cnmfe.nwb ... 66.4 MB

Reply:
@itutu-tienday
I checked its output and found that `W` and `b0` of cnmfe are quite large in size compared to `A`, `C`,...
For example, with input file is `sample_data/mouse2p_2_donotouse.tiff`:

```
>>> cnm.estimates.A.data.nbytes
1824
>>> cnm.estimates.C.data.nbytes
24000
>>> cnm.estimates.W.data.nbytes
855696
>>> cnm.estimates.b0.data.nbytes
65536
```

Reply:
@quanpython 
Understood. We will share & consult with users once.